frontend/institution/managementMembersController.js

```diff
@@ -459,33 +483,9 @@
             $mdDialog.cancel();
         };
 
-        selectEmailsCtrl.removePendingAndMembersEmails = function removePendingAndMembersEmails() {
```

Comment:
why is it removed?

Reply:
Porque iria ficar duplicado essa função, ela estava no SelectEmailsController e foi para o ManagementMembersController

Reply:
Na função sendInvite do SelectEmailsController 

selectEmailsCtrl.manageMemberCtrl.removePendingAndMembersEmails(selectEmailsCtrl.selectedEmails)